src/main/java/fi/hsl/transitdata/vehicleposition/application/VehiclePositionHandler.java

```diff
@@ -123,35 +141,37 @@ public void handleMessage(Message message) {
                         data.getTopic().getEventType() != Hfp.Topic.EventType.PAS &&
                         data.getTopic().getEventType() != Hfp.Topic.EventType.ARS &&
                         data.getTopic().getEventType() != Hfp.Topic.EventType.PDE) {
-                    log.debug("Ignoring HFP message with event type {}", data.getTopic().getEventType());
+                    log.info("Ignoring HFP message with event type {}", data.getTopic().getEventType());
                     return;
                 }
 
                 final boolean tripAlreadyTaken = !tripVehicleCache.registerVehicleForTrip(data.getTopic().getUniqueVehicleId(), data.getTopic().getRouteId(), data.getPayload().getOday(), data.getTopic().getStartTime(), data.getPayload().getDir());
 
                 if (tripAlreadyTaken && !addedTripsEnabledModes.contains(data.getTopic().getTransportMode())) {
                     //If some other vehicle was registered for the trip and the vehicle is not a bus, do not produce vehicle position
-                    log.debug("There was already a vehicle registered for trip {} / {} / {} / {} - not producing vehicle position message for {}", data.getTopic().getRouteId(), data.getPayload().getOday(), data.getTopic().getStartTime(), data.getPayload().getDir(), data.getTopic().getUniqueVehicleId());
+                    log.info("There was already a vehicle registered for trip {} / {} / {} / {} - not producing vehicle position message for {}", data.getTopic().getRouteId(), data.getPayload().getOday(), data.getTopic().getStartTime(), data.getPayload().getDir(), data.getTopic().getUniqueVehicleId());
                     return;
                 }
 
                 if (!vehicleTimestampValidator.validateTimestamp(data, message.getEventTime())) {
                     //Vehicle had invalid timestamp..
+                    log.info("Vehicle had invalid timestamp");
```

Comment:
Väliaikaisena palikkana nää on ihan kivoja, mutta hirvittää vähän että tulee liikaa lokispammia kun näitä viestejä kuitenkin tulee melko runsaasti